src/utils/data-access.ts

```diff
@@ -0,0 +1,148 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+import { PromiseResult } from 'aws-sdk/lib/request';
+import { AWSError } from 'aws-sdk/lib/error';
+import DynamoDB, { DocumentClient, QueryOutput } from 'aws-sdk/clients/dynamodb';
+import { Configurations } from './configuration';
+import * as domain from '../domain';
+import { debug } from './logger';
+
+export class DataAccess {
+  // private static instance: DataAccess;
+
+  private readonly tableName: string = Configurations.getInstance().dynamoTableName;
+
+  private docClient: DocumentClient;
+
+  public constructor() {
+    this.docClient = new DynamoDB.DocumentClient(Configurations.getInstance().dynamoParams);
+  }
+
+  // public static getInstance(): DataAccess {
```

Comment:
I think we can remove the commented out code if we don't need it

Reply:
removed